test/test_site/expected/bugs/index.html

```diff
@@ -12,8 +12,9 @@
     <link rel="stylesheet" href="..\markbind\css\bootstrap-glyphicons.min.css" >
     <link rel="stylesheet" href="..\markbind\css\github.min.css">
     <link rel="stylesheet" href="..\markbind\css\markbind.css">
+    <link rel="stylesheet" href="markbind\layouts\default\styles.css">
```

Comment:
This should have been resolved to `..\markbind\layouts\default\styles.css` because it is outside the `bugs` folder.